shared/test/mock/data.mock.js

```diff
@@ -0,0 +1,23 @@
+
+export const MORK_ROUTES = {
+	name: "MockRoute",
+	path: "/:mock1?/mock2/:mock3?",
+	url: function(payload,i18n) {
+		return `/${i18n.t('MockRoute')}/${payload._id}/success`
+	}
+}
+export const MOCK_PAYLOAD= {
+	_id:"MockValue"
+}
+export const MOCK_LOCALSTORAGE = {
+	token:"123456",
+	mockdata:JSON.stringify({
+		mockkey:"mockvalue"
+	}),
+	getItem:(value)=> MOCK_LOCALSTORAGE[value]
+}
+
+export const MOCK_STORE = {
+	session: { token: '123456',get:(key)=> this[key]},
+	location: {pathname: '/mockValue1/mock2/mockValue2'}
+}
```

Comment:
For these I like symmetry in all environments, so I usually config alias `api` for different environments. So then we can have `shared/api/test`. In test case, api returns a `Promise.resolve(data)`.

In other environments, we should wrap superagent in a Promise - this can be used in server or client.